util/vlanips.go

```diff
@@ -0,0 +1,79 @@
+/*
+Copyright 2024 Akamai Technologies, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"fmt"
+	"net/netip"
+	"slices"
+	"sync"
+)
+
+var (
+	vlanIPsMu sync.RWMutex
+	// vlanIPsMap stores clusterName and a list of VlanIPs assigned to that cluster
+	vlanIPsMap  = make(map[string]*ClusterIPs, 0)
+	vlanIPRange = "10.0.0.0/8"
```

Comment:
We are using /11 and /8 both. Not sure if thats intentional or we wanted to use just one.

Reply:
this is used to inform `netip.Prefix` what the network prefix is, and is what makes `nextIP` work - the `/11` is what gets assigned on the node - it could be much lower too